jest.config.js

```diff
@@ -0,0 +1,6 @@
+const { configure } = require('@medly/jest-config-react');
+
+module.exports = configure({
+    rootDir: './',
+    collectCoverageFrom: ['!<rootDir>/src/utils/fetch.ts']
```

Comment:
Have we moved `coverageThreshold` to package also ??

Reply:
@vitulrana yes. You can check it  here https://github.com/medly/configs/blob/f00bdffe5b9def96aecd8d24f171e96ca123b347/packages/jest-config-react/index.js#L2

Reply:
@wsameer I feel each project can have a different config, so not sure it adds value to the package because the majority of the time people will be overriding it.

Reply:
Medly has a KPI for Unit Test Coverage, which is set to 75%. We can set the global threshold to that since that's a Medly standard. What individual projects keep or modify is upto the respective team to decide.

Reply:
@wsameer Agree, we can set to 75 in the package as its organization KPI.

Reply:
Yeah I can modify it to 75.